src/config-hardware/webcams.md

```diff
@@ -0,0 +1,23 @@
+# Configuring Webcams
+
+Pop!\_OS will automatically detect most USB webcam models upon connection.
+
+---
+
+## Disabling and Enabling Webcams
+
+You can disable your webcam by pressing `Fn` + `F10`, which disables the webcam USB bus at the device level. Press `Fn` + `F10` to re-enable the webcam, and then restart any application(s) that were using the webcam at the time it was disabled.
+
+## Webcam Compatibility
+
+Pop!\_OS uses the USB Video Class (UVC) device driver for Linux. Devices supported by this driver are [listed on the UVC driver's site](http://www.ideasonboard.org/uvc/).
+
+## Installing Cheese
+
+Test your webcam functionality using [Cheese](https://github.com/GNOME/cheese). Cheese is an open-source webcam application for Linux available for installation in the [Pop!\_Shop](manage-apps/using-pop-shop.ingmd).
+
+![Install Cheese](/images/config-webcams/install-cheese.png)
+
+Use the Launcher start Cheese. Your webcam should be automatically detected.
+
+![Cheese Launched](/images/config-webcams/launch-cheese.png)
```

Comment:
I understand why you blurred yourself out, but I feel like it's a little off, especially since Cheese has effects similar to pixelation (but you're not demonstrating them.)

I'd bet we could have Aaron or someone else at the factory take a screenshot of Cheese while wearing the Tux penguin head. Would you be good with that?

Reply:
Definitely.